index.html

```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
+            </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card">
+            <div>
+              <img src="static/images/Snapshoot Portfolio 4.png" alt="">
+            </div>
+            <div>
+              <h2 class="works-title">Multi-Post Stories</h2>
+              <div class="card_details">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                     <li>Back End Dev</li>
+                     <li>2015</li>
+                </ul>
+               
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card-2">
+              <h2 class="works-title">About Myself</h2>
+              <P class="about-me">Hello I'm a software developer I can help you build a project, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded don't hesitate to contact me.</P>
+              <p class="icon">LET'S CONNECT</p>
+                <ul class="social-icons">
+                  <li>
+                    <a href="https://twitter.com/gudo_leslie"
+                      ><img class="t-social-media" src="static/images/twitter.png.png" alt="twitter"
+                    /></a>
+                  </li>
+                  <li>
+                    <a href="https://www.linkedin.com/in/leslie-gudo-b08a49255/"
+                      ><img class="l-social-media" src="static/images/Linkedin.jpeg"  alt="linkedIn"
+                      /></a> 
+                  </li>
+                  <li>
+                    <a href="#"
+                      ><img class="m-social-media" src="static/images/medium.png" alt="medium"
+                    /></a>
+                 </li>
+                 <li>
+                    <a href="https://github.com/Leeoasis"
+                    ><img class="g-social-media" src="static/images/github.png" alt="github"
+                    /></a>
+                 </li>
+                 <li>
+                    <a href="#"
+                    ><img class="v-social-media" src="static/images/Vector (4).png" alt="vector"
+                    /></a>
+                  </li>
+                </ul>
+              <button class="resume">Get my resume</button>
```

Comment:
- [ ] Kindly consider styling this button with hover and active pseudo-class with corresponding design guidelines of colors.
- The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)